lib/cache/strategies/expiration.strategy.ts

```diff
@@ -0,0 +1,63 @@
+import {IStorage} from '../storages/IStorage';
+import {AbstractBaseStrategy} from './abstract.base.strategy';
+import {IOptions, IExpiringCacheItem} from '../types';
+
+export class ExpirationStrategy extends AbstractBaseStrategy {
+  constructor(storage: IStorage) {
+    super(storage);
+  }
+
+  public async getItem<T>(key: string): Promise<T> {
+    const item = await this.storage.getItem<IExpiringCacheItem>(key);
+    if (item && item.meta && item.meta.ttl && this.isItemExpired(item)) {
+      await this.storage.setItem(key, undefined);
+      return undefined;
+    }
+    return item ? item.content : undefined;
+  }
+
+  public async setItem(
+    key: string,
+    content: any,
+    options: IOptions,
+  ): Promise<void> {
+    options = {ttl: 60, isLazy: true, isCachedForever: false, ...options};
+
+    let meta = {};
+
+    if (!options.isCachedForever) {
+      meta = {
+        ttl: options.ttl * 1000,
```

Comment:
no magic numbers, change 1000 to a constant

Reply:
added constants file